android/src/main/java/com/swmansion/rnscreens/ScreenStackFragment.kt

```diff
@@ -272,6 +559,27 @@ class ScreenStackFragment : ScreenFragment, ScreenStackFragmentWrapper {
                 super.clearFocus()
             }
         }
+
+//        override fun reactTagForTouch(touchX: Float, touchY: Float): Int {
+//            throw IllegalStateException("Screen wrapper should never be asked for the view tag")
+//        }
+//
+//        override fun interceptsTouchEvent(touchX: Float, touchY: Float): Boolean {
+//            return false
+//        }
+//
+//        override fun getHitSlopRect(): Rect? {
+//            val screen: Screen = fragment.screen
+// //            left – The X coordinate of the left side of the rectangle
+// //            top – The Y coordinate of the top of the rectangle i
+// //            right – The X coordinate of the right side of the rectangle
+// //            bottom – The Y coordinate of the bottom of the rectangle
+//            return Rect(screen.x.toInt(), -screen.y.toInt(), screen.x.toInt() + screen.width, screen.y.toInt() + screen.height)
+//        }
+
+        override fun getPointerEvents(): PointerEvents {
```

Comment:
Added following one:

```
// We set pointer events to BOX_NONE, because we don't want the ScreensCoordinatorLayout
// to be target of react gestures and effectively prevent interaction with screens
// underneath the current screen (useful in `modal` & `formSheet` presentation).
```